pkg/crc/machine/machine.go

```diff
@@ -513,16 +513,16 @@ func (*client) IP(ipConfig IPConfig) (string, error) {
 	return ip, nil
 }
 
-func (*client) Status(statusConfig ClusterStatusConfig) (ClusterStatusResult, error) {
+func (*client) Status(statusConfig ClusterStatusConfig) (*ClusterStatusResult, error) {
 	libMachineAPIClient, cleanup, err := createLibMachineClient(false)
 	defer cleanup()
 	if err != nil {
-		return statusError(statusConfig.Name, "Cannot initialize libmachine", err)
+		return statusError("Cannot initialize libmachine", err)
```

Comment:
This would feel more 'natural' as `return nil, statusError(...)`

Reply:
I removed all statusError/etc functions.